src/index.d.ts

```diff
@@ -0,0 +1,256 @@
+// Type definitions for bonescript 0.6
+// Project: https://github.com/jadonk/bonescript
+// Definitions by: Troy W. <https://github.com/troywweber7>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as SerialPort from 'serialport';
+import * as VError from 'verror';
+
+// OBJECTS
+export const bone: BoneObj;
+
+// FUNCTIONS - TODO obliterate all "any" types where possible! --- TWW
+// https://github.com/jadonk/bonescript#system
+
+export function getPlatform(callback?: (error: ErrorType, platform: any) => void): any;
```

Comment:
Is there a reason that 'platform' is of type 'any' and not a particular structure?